### PR TITLE
Target net6.0 for linker

### PR DESCRIPTION
**Pull request by vitek-karas**
Fixes https://github.com/mono/linker/issues/2118

### CODE REVIEW

File: src/ILLink.Tasks/ILLink.Tasks.csproj [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <!-- Keep these in sync with _ILLinkTasksTFM in Sdk.props. -->
```
Comment (sbomer) [inline src/ILLink.Tasks/ILLink.Tasks.csproj:3]: Might be worth updating this comment.

**Review by sbomer — APPROVED** [review]

**Review by agocke — APPROVED** [review]
LGTM. Does this change anything since the SDK usually forces everyone to net6.0 in publishing anyway?

**Comment by vitek-karas** [comment]
> Does this change anything since the SDK usually forces everyone to net6.0 in publishing anyway?

Not really - the only scenario affected is if you run linker directly. Nobody does that - except us when debugging the linker.

**Comment by lewing** [comment]
This seems to be causing issues dotnet/sdk#19655 and dotnet/runtime#57203

**Comment by vitek-karas** [comment]
Working on it...

**Comment by mateoatr** [comment]
I'm taking a look. This appears to be caused by the analyzer not taking into account the `SuppressTrimAnalysisWarnings`, `ILLinkWarningLevel` properties.

**Comment by mateoatr** [comment]
> I'm taking a look. This appears to be caused by the analyzer not taking into account the `SuppressTrimAnalysisWarnings`, `ILLinkWarningLevel` properties.

Ok, the error was about wrong path. Thanks for clarification @vitek-karas.

**Pull request merged**